router/worker.go

```diff
@@ -1030,7 +1033,7 @@ func (w *worker) postStatusOnResponseQ(respStatusCode int, destinationJob *types
 			worker:     w,
 			job:        destinationJobMetadata.JobT,
 			status:     status,
-			payload:    inputPayload,
+			payload:    payload,
```

Comment:
If it's success, should we keep the `destinationJobMetadata.JobT.EventPayload`. we don't surface it on UI and also, given we might capture huge payloads if we start capturing the router output for successes too?